MonstieWash/Assets/Scripts/UI/NagivationUI.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Generates nav UI. Finds scene camera, sets it to Canvas render camera.
+/// </summary>
+public class NavigationUI : MonoBehaviour
+{
+    private List<GameScene> gameScenes;
+    [SerializeField] private GameObject buttonTile;
+    [SerializeField] private GameObject navPanel;
+
+    [SerializeField] private float tilePadding = 1.25f; //Seems like a nice number. Means that the UI doesn't overlap with the bag on Mimic.
```

Comment:
### 🗒️ **Convention** | _This is for maintaining our coding conventions._

Just the `m_` prefix and ordering.

```suggestion
    [SerializeField] private GameObject buttonTile;
    [SerializeField] private GameObject navPanel;
    [SerializeField] private float tilePadding = 1.25f; //Seems like a nice number. Means that the UI doesn't overlap with the bag on Mimic.
    
    private List<GameScene> m_gameScenes;
```